tests/foreman/ui/test_discoveredhost.py

```diff
@@ -319,21 +281,3 @@ def test_positive_update_default_taxonomies(session, discovery_org, discovery_lo
         assert set(host_names) == {value['Name'] for value in values}
         values = session.dashboard.read('DiscoveredHosts')
         assert len(values['hosts']) == 2
-
-
-@pytest.mark.stubbed
-@pytest.mark.tier3
-def test_positive_reboot(session, provisioning_env):
```

Comment:
is this also being covered in tests above?

Reply:
We are covering reboot and reboot_all endpoints in the API module